host/tests/L3_TestCases/dsCompositeIn/dsCompositeInHelperClass.py

```diff
@@ -144,7 +142,7 @@ def testPrepareFunction(self):
         self.testRunPrerequisites()

```

Comment:
is possible to use the player class do the prerequisites rather than the reading from device config?

Reply:
player/gsteamer Is not required for CompositeIn and not inheriting utPlayer class also..
These prerequisite is required for playing a video from compositeIn source device...

Reply:
Removed prerequisite from deviceConfig and used is the utplayerConfig